### ISSUE TITLE
Content Language

### ISSUE THREAD

**Issue by weissmi1** [MicrosoftDocs/Viva#653]
[Enter feedback here]

can you please update the documentation for the SharePoint content language? In another section, I found the information, that the custom field "ContentLanguage" should be used to tag the content with the right language.

Can you maybe also update the format accepted for this field?

For German content, should we use "Deutsch (Deutschland)" ?

many Thanks
Michael

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: db22de2a-0ec5-d71b-96f6-617585e0489b
* Version Independent ID: db22de2a-0ec5-d71b-96f6-617585e0489b
* Content: [Use Microsoft 365 Groups permissions with SharePoint content in Viva Learning](https://learn.microsoft.com/en-us/viva/learning/sharepoint-permissions)
* Content Source: [Viva/learning/sharepoint-permissions.md](https://github.com/MicrosoftDocs/viva/blob/public/Viva/learning/sharepoint-permissions.md)
* Service: **viva**
* Sub-service: **viva-learning**
* GitHub Login: @bhaswatic
* Microsoft Alias: **bhaswatic**

**Comment by get-itips** [comment]
Hello @weissmi1 

How are you?

> In another section, I found the information, that the custom field "ContentLanguage" should be used to tag the content with the right language.

Can you please point me to that article? Thanks

**Comment by weissmi1** [comment]
Yes you can find this information in https://learn.microsoft.com/en-us/viva/learning/language-preferences
